base_external/package/aesd-assignments/aesd-assignments.mk

```diff
@@ -1,13 +1,5 @@
-
-##############################################################
```

Comment:
@AjiteshBatraCU in this case I'd suggest adding the two files you need to a rootfs overlay instead of adding a dependency to your github repo and final project application.

Reply:
@VatsalSheth we should be able to remove aesd-assignments.mk and dependency on Ajitesh's final project application repo